protocol-units/execution/opt-executor/src/executor/transaction_pipe.rs

```diff
@@ -28,6 +28,13 @@ impl From<anyhow::Error> for TransactionPipeError {
 }
 
 impl Executor {
+
+	pub async fn gc_mempool(&self) -> Result<(), TransactionPipeError> {
+		let mut core_mempool = self.core_mempool.write().await;
+		core_mempool.gc();
```

Comment:
Gc can be dangerous because it locks all the mempool. Perhaps we should add a log to instrument how long it takes to have an idea and identify too long lock.

Reply:
I can add a span to the tracing instrumentation later on.

Reply:
Even better, add a span now and we'll add target information later.

Reply:
@musitdev To make this compatible with the lockless mempool, I can just track the last time `gc` was run in the mempool loop.